clusterloader2/pkg/measurement/common/probes/manifests/metricsServer/metrics-server-prober-rbac.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I don't understand this...

k8s doesn't support list containing multiple types of objects...

I still don't understand why you're touching it - what didn't work with the original file?

Reply:
the issue is here
https://github.com/kubernetes/perf-tests/blob/e6c9306927161ab03460f2d7f0d284e78a732c2e/clusterloader2/pkg/framework/framework.go#L275
According to the original file, only the first object can be parsed from the file, and the following cannot be parsed.

Reply:
and I have observed that other manifest files have only one resource.

Reply:
OK - that makes sense.